README.md

```diff
@@ -154,3 +154,33 @@ class User < ApplicationRecord
          }
 end
 ```
+
+#### Intercept request/response
+
+You can register before- and after- request hooks in CastleDevise.
+
+```ruby
+CastleDevise.configure do |config|
+  # Add custom properties to the request but only when sending
+  #   requests to the Risk endpoint
+  # meth - Castle API endpoint (eg. :risk, :filter, :log)
```

Comment:
wouldn't it be better to name it `method_name`, `endpoint`, `endpoint_name`, `action`?

Reply:
I like `action`, will rename :+1: